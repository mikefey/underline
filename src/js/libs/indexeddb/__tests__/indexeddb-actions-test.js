import test from 'ava';
import {
  dbReady,
} from './../indexeddb-actions';

test('dbReady action', (t) => {
  t.snapshot(dbReady());
});
