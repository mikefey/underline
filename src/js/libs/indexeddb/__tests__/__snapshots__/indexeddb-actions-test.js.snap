exports[`dbReady action 1`] = `"{\"type\":\"DB_READY\"}"`;
